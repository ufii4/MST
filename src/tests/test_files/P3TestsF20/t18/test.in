n 4
i 0;1;500
i 0;2;501
i 0;3;502
degree 0
degree 1
degree 2
degree 3
edge_count
mst
i 1;0;503
i 1;2;504
i 1;3;505
i 2;0;506
i 2;1;507
i 2;3;508
i 3;0;509
i 3;1;510
i 3;2;511
degree 0
degree 1
degree 2
degree 3
edge_count
mst
exit
