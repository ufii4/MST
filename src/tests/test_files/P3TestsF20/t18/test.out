success
success
success
success
degree of 0 is 3
degree of 1 is 1
degree of 2 is 1
degree of 3 is 1
edge count is 3
mst 1503.00
success
success
success
success
success
success
success
success
success
degree of 0 is 3
degree of 1 is 3
degree of 2 is 3
degree of 3 is 3
edge count is 6
mst 1518.00
